docs/source/app-implementing-new.rst

```diff
@@ -0,0 +1,117 @@
+App Integration For New Service
```

Comment:
`Adding a New App`

Reply:
@mime-frame can this be `Creating a New App` since 'adding' is ambiguous to me and could be conceived as 'configuring'

Reply:
Good call, I like your suggestion